src/tests/multihost/ad/test_dyndns.py

```diff
@@ -0,0 +1,537 @@
+        :expectedresults:
+          1. Client joins domain
+          2. Hostname does not resolve correctly on bogus interface
+          3. IP does not resolve correctly on bogus interface
+        """
+        adjoin(membersw='adcli')
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        dns = ADDNS(multihost.ad[0])
+        hostname = multihost.client[0].run_command('hostname').stdout_text.rstrip()
+        domain = multihost.ad[0].domainname
+        ip = multihost.client[0].ip
+        dns.del_record(hostname)
+        dns.del_record(ip)
+
+        domain_section = 'domain/{}'.format(domain)
+        sssd_params = {'dyndns_iface': 'non_existent'}
+        client.sssd_conf(domain_section, sssd_params)
+        client.clear_sssd_cache()
+
+        assert dns.find_a(hostname) is not True
+        assert dns.find_ptr(hostname, ip) is not True
+
+    @staticmethod
+    def test_0006_check_with_dyndns_refresh_interval(multihost, adjoin, extra_network, extra_interface):
+        """
+        :title: IDM-SSSD-TC: ad_provider: dyndns: check with dyndns refresh interval
+        :id: 54765188-b21f-4ba1-890b-953eec9b696d
+        :steps:
+          1. Join client to AD domain and delete all client DNS records
+          2. Create extra interface and assign IP
+          3. Set refresh interval and dyndns iface in sssd.conf
+          4. Perform DNS lookup for hostname on extra interface
+          5. Perform DNS lookup for IP on extra interface
+          6. Change IP on extra interface
+          7. Let SSSD update DNS
+          8. Perform DNS lookup for hostname on extra interface
+          9. Perform DNS lookup for IP on extra interface
+        :expectedresults:
+          1. Client joins domain
+          2. Should succeed
+          3. Should succeed
+          4. Hostname resolves correctly on extra interface
+          5. IP resolves correctly on extra interface
+          6. New IP address is assigned for extra interface
+          7. SSSD updates DNS
+          8. Hostname resolves correctly on extra interface with new IP
+          9. IP resolves correctly on extra interface with new hostname
+        """
+        (extra_interface, extra_ip) = extra_interface
+        extra_ip_after_refresh = '192.168.1.' + str(random.randrange(2, 255))
+        while extra_ip == extra_ip_after_refresh:
+            extra_ip_after_refresh = '192.168.1.' + str(random.randrange(2, 255))
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        dns = ADDNS(multihost.ad[0])
+        hostname = multihost.client[0].run_command('hostname').stdout_text.rstrip()
+        domain = multihost.ad[0].domainname
+        ip = multihost.client[0].ip
+        dns.del_record(hostname)
+        dns.del_record(ip)
+
+        adjoin(membersw='adcli')
+        domain_section = 'domain/{}'.format(domain)
+        sssd_params = {'dyndns_refresh_interval': '81', 'dyndns_iface': extra_interface}
```

Comment:
minimum `dyndns_refresh_interval` value is 60, why not use 60 here and `time.sleep(60)` ?

Reply:
Disregard, I see it explained in the next test comments